### DISCUSSION TITLE
Spatial Single Cell Gene Expression MERFISH Visualization

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#6061] [Show and tell]
We built this interactive spatial single cell gene expression visualization using deck.gl

https://twitter.com/vizgen_inc/status/1422897728808423430?s=19

Thanks for all the help on discussion boards!

![Screenshot_20210804-100609](https://user-images.githubusercontent.com/8352840/128195379-98c3cef3-0f6b-4e78-940b-ce3cca20bc26.png)

**Comment by ibgreen** [comment]
@cornhundred Congrats on the release! This looks amazing and would be an awesome addition to the [deck.gl show cases](https://deck.gl/showcase).

(I assume you are already collaborating with the [viv](https://github.com/hms-dbmi/viv) team as they are also using deck for biological visualizations?  @ilan-gold @manzt)

PPS - Would you like to present your work to the community in the next deck.gl open governance meeting? If so, ping me on the deck.gl slack.

**Comment by cornhundred** [comment]
Thanks @ibgreen! That would be great to have our example added to the deck.gl showcase page and I would be very happy to present our work at the community governance meeting. I'll message you on the unfolded slack. Is there a deck.gl specific slack?

We are not currently collaborating with the viv team but have chatted on GitHub a bit :)

**Reply by kylebarron** [reply]
> Is there a deck.gl specific slack?

https://deck.gl/docs/get-started/learning-resources#community

**Reply by cornhundred** [reply]
thanks! @kylebarron

**Reply by ibgreen** [reply]
You should be able to open a PR to add your example to the showcases. Do you need pointers?

**Reply by cornhundred** [reply]
Thanks @ibgreen I made the pull request here #6067. Please let me know if anything needs to be modified.
